added troubleshooting.md page and table_of_contents.html needed for T…

gyrusdentatus opened the pull request:
…able of contents in the troubleshooting.md to work

futurechimp commented:
Sorry I missed this, I had a mail filter in place to route the daily tidal wave of Github notifications and this got caught up in it. I've fixed this in another commit now, but thanks for the pull request and we'll get to it quicker next time.

Commit:
added troubleshooting.md page and table_of_contents.html needed for Table of contents in the troubleshooting.md to work

futurechimp commented on content/run-nym-nodes/troubleshooting.md:
```diff
@@ -0,0 +1,283 @@
+---
+title: "Troubleshooting FAQ"
+weight: 30
```
Hey if you change this to 40 it'll drop to the bottom, and I'll merge the pull :)

gyrusdentatus committed:
Update troubleshooting.md

changed the weight to 40

gyrusdentatus reviewed (commented):
hope it's okay now.

gyrusdentatus committed:
Update troubleshooting.md

edited a few typos, changed "I have seen..." and "here is an example of MY test run..." and directory tree structure of the .nym/ in "where to find my keys and config" section of the document

gyrusdentatus reviewed (commented):
should be now ready to go public

futurechimp commented on content/run-nym-nodes/troubleshooting.md:
```diff
@@ -0,0 +1,282 @@
+
+### Docker images
+
+We currently do not have an official Docker image, although there exist at least two community images. Decide for yourself whichever suits your needs better. 
```
Two Docker images are mentioned here, but then the following list has only one - is there one missing?

futurechimp commented on content/run-nym-nodes/troubleshooting.md:
```diff
@@ -0,0 +1,282 @@
+
+There are multiple ways how to do that.
+
+### To check from the machine your node is running on:
```
Do people really need 4 ways to do this? `telnet` will work from any machine (local or remote). Maybe it would be better to just give one tool and list what the correct output should be? And what different kinds of errors mean? 

E.g. if I do: 

```
dave@constructiveproof:~$ telnet constructiveproof.com 80
Trying 85.159.210.216...
Connected to constructiveproof.com.
Escape character is '^]'.
```

That's a win. 

If I do: 

```
dave@constructiveproof:~$ telnet constructiveproof.com 12345
Trying 85.159.210.216...
telnet: Unable to connect to remote host: Connection refused
```

There is nothing running on port 12345. So either the application is not running, or there is a firewall problem, or the IP is wrong. 

If you try a telnet and the connection hangs without doing anything, usually that would mean that everything is up and running but the firewall is dropping traffic on that port, so there's a firewalling issue.

futurechimp commented on content/run-nym-nodes/troubleshooting.md:
```diff
@@ -0,0 +1,282 @@
+
+First of all, your ISP has to be IPv6 ready. Sadly, in 2020, most of them are not and you won't get an IPv6 address by default from your ISP. Usually it is a extra paid service or they simply don't offer it. 
+
+Before you begin, check if you have IPv6 [here](https://test-ipv6.cz/). If not, then don't waste your time to run a node which won't ever be able to mix any packet due to this limitation. Call your ISP and ask for IPv6, there is a plenty of it for everyone!
```
This one might accidentally discourage people in an undesirable way - I think it needs to be more clear that this applies only to people who are attempting to run their nodes out of their homes. In my own case, I can't run a mixnode from the house, because my ISP doesn't support IPv6. But I can happily run one from my hosting provider (Linode).

futurechimp committed:
A few more spelling checks and code fences

mfahampshire commented:
closing - out of date / already resolved
